src/main/resources/application.yml

```
@@ -36,4 +36,7 @@ jwt:
     accessTime: ${JWT_ACCESS_TIME}
     refreshTime: ${JWT_REFRESH_TIME}
 server:
-  port: 8081
\ No newline at end of file
+  port: 8081
+aes:
+  secret-key: 0123456789abcdef0123456789abcdef
+  init-vector: abcdef9876543210
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)